examples/quest-2-step-options/qwiklabs.yaml

```diff
@@ -0,0 +1,89 @@
+entity_type: Quest
+schema_version: 1
+
+# Quest Attributes
+id: my-awesome-quest
+
+tags: [sample, life-changing, gcp]
+logo: ./quest-logo.png
+default_locale: en
+
+title:
+  locales:
+    en: GCP Survey Quest
+
+description:
+  locales:
+    en: Get a taste of what GCP has to offer.
+
+objectives:
+  locales:
+    en: |
+      <p>This course will teach people</p>
+      <ul><li>The power of frienship</li></ul>
+
+audience:
+  locales:
+    en: |
+      <p>This course is intended for people who</p>
+      <ul><li>Do not understand the power of frienship</li></ul>
+
+prerequisites:
+  locales:
+    en: <p>A firm understanding of interpersonal interactions is recommended but not required.</p>
+
+suggested_duration:
+  locales:
+    en: 1 day
+
+
+resources:
+  - type: video
+    ref_id: intro-video
+    locales:
+      en:
+        title: Welcome to GCP!
+        uri: https://youtu.be/dNW0B0HsvVs
+      es:
+        title: ¡Bienvenido a GCP!
+        uri: https://youtu.be/dNW0B0HsvVs
+  - type: video
+    ref_id: outro-video
+    locales:
+      en:
+        title: Wasn't that fun!
+        uri: https://youtu.be/dNW0B0HsvVs
+      es:
+        title: Eso fue divertido, ¿verdad?
+        uri: https://youtu.be/dNW0B0HsvVs
+
+steps:
+  - content: $intro-video
```

Comment:
I immediately knew this syntax was referring to a resource defined in the bundle (and I like it), but I'm concerned that special characters will raise the barrier to entry for some people.

Reply:
Honestly, I agree.

I'm looking to find a balance for making both resources and labs easy to reference without making it a mess to interpret.

Reply:
How about this...?

```yaml
steps:
  - resource: intro-video
  - lab: intro-to-gcp
  ...
```

Reply:
How about `resources/intro-video` instead (where `resources` is now a reserved name and nobody can make a library called `resources` now).?